jobs/build/ocp4/build.groovy

```diff
@@ -240,6 +243,33 @@ def planBuilds() {
         currentBuild.description += "error during plan builds step:\n${err.getMessage()}\n"
         throw err
     }
+
+    // check whether the group should be built right now
```

Comment:
This is OK as far as it goes, but can we extract this to commonlib and use it for the `sweep` job as well? That's another thing we don't want running during release work.

Reply:
I put it in buildlib with the same end goal - since it was doozer / build based.